### PR TITLE
October 2023 Meetup Post

### PR DESCRIPTION
**Pull request by AngelOnFira**
@noxim @puzzled-squid if you could check this out that would be great!

### CODE REVIEW

File: content/blog/rust-gamedev-meetup-31/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+[youtube-stream]: https://www.youtube.com/@RustGameDevelopment
+[twitch-stream]: https://www.twitch.tv/RustGameDev
+
+## Talks
+
+### [Games, but no video: Embedded trinkets in Rust] | [@Noxim]
+
+![Hardware](hardware.jpg)
+
+> An overview of making games with constrained hardware
```
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-31/index.md:33]: @noxim What would go better as a talk description?

File: content/blog/rust-gamedev-meetup-31/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+[PAC + HAL]: https://github.com/ch32-rs
+[More details]: https://noxim.xyz/blog/rust-ch32v003/
+[Games, but no video: Embedded trinkets in Rust]: https://www.youtube.com/live/MadryxOwjb8?si=erY49m_5_HDFkrpf&t=694
+[@Noxim]: https://github.com/Noxime
+
+### [Tunnet] | [@puzzled_squid]
+
+![tunnet title](tunnet.jpg)
+
+> A short game where you wake up as a robot technician in an underground complex
```
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-31/index.md:55]: @puzzled-squid What would go better as a talk description?

**Comment by AngelOnFira** [comment]
Also, let me know if there should be more links, or if some should be removed :)

File: content/blog/rust-gamedev-meetup-31/index.md (hunk 2 above)
Comment (puzzled-squid) [reply content/blog/rust-gamedev-meetup-31/index.md:55]: I would phrase it this way: "A short game where you design, build, and optimize a computer network in an undergound complex"

**Comment by puzzled-squid** [comment]
> Also, let me know if there should be more links, or if some should be removed :)

 It looks good to me. Thanks a lot!

File: content/blog/rust-gamedev-meetup-31/index.md (hunk 2 above)
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-31/index.md:55]:
```suggestion
> A short game where you design, build, and optimize a computer network in an underground complex
```

File: content/blog/rust-gamedev-meetup-31/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- [YouTube trailer]
+- [Website]
+
+![tunnet title](tunnet-network.jpg)
+
+[itch.io page]: https://puzzled-squid.itch.io/tunnet
+[YouTube trailer]: https://www.youtube.com/watch?v=6gxytilYoOM
+[Website]: https://puzzledsquid.xyz/
+[Tunnet]: https://www.youtube.com/live/MadryxOwjb8?si=vZrhqUszuv9qvG9t&t=2878
+[@puzzled_squid]: https://github.com/puzzled-squid
```
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-31/index.md:67]:
```suggestion
[@puzzled_squid]: https://github.com/puzzled-squid

```

File: content/blog/rust-gamedev-meetup-31/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+[PAC + HAL]: https://github.com/ch32-rs
+[More details]: https://noxim.xyz/blog/rust-ch32v003/
+[Games, but no video: Embedded trinkets in Rust]: https://www.youtube.com/live/MadryxOwjb8?si=erY49m_5_HDFkrpf&t=694
+[@Noxim]: https://github.com/Noxime
+
+### [Tunnet] | [@puzzled_squid]
+
+![tunnet title](tunnet.jpg)
+
+> A short game where you design, build, and optimize a computer network in an underground complex
```
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-31/index.md:55]:
```suggestion
> A short game where you design, build, and optimize
> a computer network in an underground complex
```

**Pull request merged**